Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Base Python image for container
+FROM python:3.7
+
+# Set unbuffered output to make sure all logs are printed and not stuck in buffer
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update
+RUN mkdir -p /c3po
+
+# Copy and install requirements
+ADD requirements /c3po/requirements
+RUN pip install --upgrade pip
+RUN pip install -r /c3po/requirements/common.txt && pip install -r /c3po/requirements/dev.txt
+
+# Copy emplate config file
+COPY config.template.ini /c3po/config.ini
+
+ADD . /c3po/
```

Comment:
Not an issue. Any specific reason to copy only requirements first and then the whole directory?

Reply:
Nope, no particular reason. I was reading about some good Docker practices and it mentioned that copying dependencies and installing dependencies first is a good idea since in the future when I want to rebuild certain changes in the repo, it won't reinstall dependencies (Since the copying happens in a later step). 
Putting the dependencies first will cache and reuse even if files change.